migtests/scripts/functions.sh

```diff
@@ -947,6 +947,18 @@ compare_sql_files() {
 	sed -i -E 's#ALTER action DISABLE RULE#ALTER TABLE DISABLE RULE#g' "$normalized_file1"
 	sed -i -E 's#ALTER action ALTER COLUMN ... SET#ALTER TABLE ALTER COLUMN#g' "$normalized_file1"
 
+	# Replace the PostGIS "extension not available" message with the "could not open control file" message for 2.25
+	sed -i -E 's#ERROR: extension "postgis" is not available \(SQLSTATE 0A000\)#ERROR: could not open extension control file "PATH_PLACEHOLDER/postgis.control": No such file or directory (SQLSTATE 58P01)#g' "$normalized_file1"
+
+	# Changes required for 2.25 in the mgi schema. We still run Jenkins tests on 2024.2
```

Comment:
remove these comments if not needed?

Reply:
These are fixes required for when we switch the Jenkins pipelines to these versions (2.25+). So have commented them for now to reduce unnecessary normalisation.